run_migrations.py

```
@@ -57,20 +58,67 @@ def is_first_time():
 
+async def database_tables_exist():
+    """Check if the 'aerich' table exists in the PostgreSQL database (Supabase)."""
+    try:
+        # Initialize Tortoise connection
+        await Tortoise.init(config=TORTOISE_ORM)
+
+        connection = Tortoise.get_connection("default")
+
+        # Execute the query to check for 'aerich' table
+        rows, _ = await connection.execute_query(
+            "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname = 'public' AND tablename = 'aerich';"
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)